monitoring/monitoring_client.py

```diff
@@ -97,12 +95,14 @@ def _main():
             except requests.exceptions.RequestException:
                 alert("Exception when trying to reach pi@{}".format(ip))
             if r:
-                r_json = r.json()
+logs                r_json = r.json()
```

Comment:
have you tested this? because it seems like a bug. do not submit broken code to a PR.